

try

   notafun(1)
# catch
#    println("sorry")

finally
   print("finalprint")
end # try
